src/ergw_aaa_session.erl

```diff
@@ -177,6 +177,7 @@ init([Owner, SessionOpts]) ->
 
     App = ergw_aaa:get_application(AppId),
     OriginHost = maps:get('Origin-Host', App, net_adm:localhost()),
+    ?LOG(debug, "My origin host is ~p", [OriginHost]),
```

Comment:
too much logging for my taste, but if you really want to keep it, make it impersonal at least, e.g. "DIAMETER Origin-Host set to...."

Reply:
Ok, since the diameter session id will use this info and it will be externally available, I can remove this.